alf/algorithms/mi_estimator.py

```diff
@@ -134,17 +135,19 @@ def __init__(self,
         self._y_spec = y_spec
         if model is None:
             if estimator_type == 'ML':
-                model = TFAEncodingNetwork(
+                model = EncodingNetwork(
                     name="MIEstimator",
                     input_tensor_spec=x_spec,
                     fc_layer_params=fc_layers,
-                    preprocessing_combiner=NestConcatenate(axis=-1))
+                    preprocessing_combiner=NestConcat(dim=-1))
```

Comment:
last_activation=math_ops.identity

Reply:
The TF version uses TFAEncodingNetwork. There is not special handling for last layer.

Reply:
Oh I thought it's our encoding network.